media-gfx/qview/additional-files/qview.rdef

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
The "qview.rdef" isn't needed any longer. We now have  "qview.rdef.in" with all the info.